apps/ts_example/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "@canonical/typescript-config-base",
   "compilerOptions": {
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "types": ["node"]
```

Comment:
This shouldn't need `node` anymore, since we have removed the node-specific type depencies from the utils package.